src/routes/home/HomeScreen.tsx

```diff
@@ -56,6 +60,21 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
     })
   }
 
+  const navigateToNextExercise = async (item: Discipline): Promise<void> => {
+    getNextExercise(item)
+      .then(async nextExerciseData => {
+        const [disciplineId, exerciseKey] = nextExerciseData
+        const documents = await loadDocumentsById(disciplineId)
+        navigation.navigate(EXERCISES[exerciseKey].nextScreen, {
```

Comment:
If you navigate to `EXERCISES[exerciseKey].nextScreen`, `getNextExercise` does not really return the next exercise, does it?

Reply:
getNextExercise returns the exerciseKey of the nextExercise? I think I do not get your question?

Reply:
But why do we need `.nextScreen` then? Isn't it the next exercise of the exercise with exercise key?

Reply:
Ah, oke, I see where your confusion comes from. Naming here is really shitty, as "nextScreen" is just the screen of this exercise. I will refactor this in a separate PR.